haystack/nodes/prompt/prompt_node.py

```diff
@@ -232,6 +261,19 @@ def __init__(
         devices: Optional[List[Union[str, torch.device]]] = None,
         **kwargs,
     ):
+        """
+        Creates an instance of HFLocalInvocationLayer used to invoke local Hugging Face models.
+
+
+        :param model_name_or_path: The name or path of the underlying model.
+        :param max_length: The maximum length of the output text.
+        :param use_auth_token: The token to use as HTTP bearer authorization for remote files.
+        :param use_gpu: Whether to use GPU for inference.
+        :param device: The device to use for inference.
+        :param kwargs: Additional keyword arguments passed to the underlying model.
```

Comment:
Could we add the comments on lines 287-290 to the docstrings here so users can better understand how kwargs are handled?

Reply:
@sjrl I think I have addressed your comments. See https://github.com/deepset-ai/haystack/pull/3821/commits/f549bac1e1b3ee0f12f64f6098b877d3c95f81c4 for more details